web/src/main/webapp/table.ts

```diff
@@ -870,9 +879,31 @@ export class TableView extends RemoteObject
                     cell.classList.add("missingData");
                     cell.textContent = "missing";
                 } else {
-                    cell.textContent = TableView.convert(row.values[dataIndex], cd.kind);
+                    let cellValue : string = TableView.convert(row.values[dataIndex], cd.kind);
+                    cell.textContent = cellValue;
+                    if (cd.kind == "String" || cd.kind == "Json" || cd.kind == "Category" || cd.kind == "Integer") {
+                        cell.oncontextmenu = e => {
+                            e.preventDefault();
+                            if (this.contextMenu != null) {
+                                this.contextMenu.remove();
+                            }
+                            this.contextMenu = new ContextMenu([
+                                {text: "Filter for " + cellValue, action: () => this.equalityFilter(cd.name, cellValue) },
+                                {text: "Filter for not " + cellValue, action: () => this.equalityFilter(cd.name, cellValue, true) }
+                            ]);
+
+                            document.body.appendChild(this.contextMenu.getHTMLRepresentation());
```

Comment:
Fixed this! There's now a single context menu (per TableView) that is now hidden/shown instead of removed/created.